Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+# This Dockerfile is for HSSM
+# The buid from the base of scipy-notebook, based on python 3.11
+
+FROM jupyter/scipy-notebook:python-3.11
+
+LABEL maintainer="Hu Chuan-Peng <hcp4715@hotmail.com>"
+
+USER root
+
+RUN apt-get update -y && \
+  apt-get upgrade -y && \
+  apt-get install -y apt-utils && \
+  apt-get install -y build-essential&& \
+  apt-get install -y gcc && \
+  apt-get install -y g++ && \
+  apt-get install -y gfortran && \
+  rm -rf /var/lib/apt/lists/*
+
+USER $NB_UID
+
+RUN conda config --add channels  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
```

Comment:
This might be an issue for most of the users. Any reason why we cannot use `conda-forge`?

Reply:
In our experience, we encountered an issue with the pip-installed pytensor not being able to find the blas library, resulting it falling back to the numpy C-API and much slower performance overall. That's why we worked to put hssm on PyPI. Is there any issue like this with this docker image?

Reply:
I'm not sure if we installed pytensor correctly. But using the following code, I found that pytensor should recognize the blas library. 

```python
import pytensor
print(pytensor.config.blas__ldflags)
# -L/opt/conda/lib -llapack -lblas -lcblas -lm -Wl,-rpath,/opt/conda/lib
```

Please let me know if there are other better testing methods to ensure that pytensor is installed correctly